src/CyBy/Draw/MoleculeCanvas.idr

```diff
@@ -20,6 +20,20 @@ import Text.SVG
 %language ElabReflection
 %hide Language.Reflection.TTImp.Mode
 
+
+-- Record pairing graph with one of its node's index
+record DepNode where
```

Comment:
This needs to be `public export` otherwise we can't pattern match on `Mode`. The comment should therefore be a docstring.